Description: Updated links to source code
Author: Araq <rumpf_a@web.de>
Applied-Upstream: 1af20535d8105287aa8c73ff1a90e102c4590ffb
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/tools/nimweb.nim
+++ b/tools/nimweb.nim
@@ -48,7 +48,7 @@
   c.logo = ""
   c.ticker = ""
   c.vars = newStringTable(modeStyleInsensitive)
-  c.gitRepo = "https://github.com/Araq/Nim/tree"
+  c.gitRepo = "https://github.com/nim-lang/Nim/tree"
   c.gitCommit = "master"
   c.numProcessors = countProcessors()
   # Attempts to obtain the git current commit.
